projects/igniteui-angular/src/lib/directives/tooltip/tooltip-target.directive.ts

```diff
@@ -300,21 +303,40 @@ export class IgxTooltipTargetDirective extends IgxToggleActionDirective implemen
         this._overlayDefaults.closeOnOutsideClick = false;
         this._overlayDefaults.closeOnEscape = true;
 
+        this.target.opening.pipe(takeUntil(this.destroy$)).subscribe(() => {
+            if (this.target.tooltipTarget === this) {
+                document.addEventListener('touchstart', this.onDocumentTouchStart = this.onDocumentTouchStart.bind(this), { passive: true });
```

Comment:
It's wrong to do `this.onDocumentTouchStart = this.onDocumentTouchStart.bind(this)`
inside the event handler. You will be creating a brand new instance each time it opens. You can bind just once in the constructor or on init.